Documentation/building/unix-instructions.md

```diff
@@ -1,16 +1,90 @@
 Building CoreFX on FreeBSD, Linux and OS X
 ==========================================
 
-Building CoreFX is pretty straightforward. Clone the repo and run the build script.  For specific steps to build on Ubuntu, [navigate to the end of this document](#steps-to-build-on-ubuntu).
+The CoreFX build has two logical components, the native build which produces the
+"shims" (which provide a stable interface between the OS and managed code) and
+the managed build which produces the MSIL code and nuget packages that make up
+CoreFX.
+
+The native component should be buildable on any system, but the managed
+components require a version of the .NET Core CLI (which the build will
+download) so managed components can only be built on a subset of distros.
+
+### Prerequsites (native build)
+
+The native build produces shims over libc, openssl, libcurl and libz. The build
+system uses CMake (2.8.12 or higher) to generate Makefiles using clang (3.5 or
+higher).  The build also uses git for generating some version information.
```

Comment:
It seems to work without it but I didn't do much testing beyond running the build and seeing there were no errors. I'm more concerned about being able to build outside of a git repository than being able to build without git installed. But, it seems to work regardless.